src/envoy/http/authn/http_filter.h

```diff
@@ -18,9 +18,18 @@
 #include "authentication/v1alpha1/policy.pb.h"
 #include "common/common/logger.h"
 #include "server/config/network/http_connection_manager.h"
+#include "src/envoy/http/authn/authenticator_base.h"
+#include "src/envoy/http/authn/filter_context.h"
 
 namespace Envoy {
 namespace Http {
+namespace Istio {
+namespace AuthN {
+
+enum State { INIT, PROCESSING, COMPLETE, REJECTED };
+
+}  // namespace AuthN
+}  // namespace Istio
```

Comment:
AuthenticationFilter should be inside Istio::AuthN namespace too.

Reply:
Done. Btw, we are lacking consitency here. Jwt doesn't have a separate namespace, and mixer is simply Mixer.